plugins/common/cookie/cookie.go

```diff
@@ -20,7 +20,7 @@ type CookieAuthConfig struct {
 	URL    string `toml:"cookie_auth_url"`
 	Method string `toml:"cookie_auth_method"`
 
-	Headers map[string]string `toml:"cookie_auth_headers"`
+	Headers map[string]*config.Secret `toml:"cookie_auth_headers"`
```

Comment:
This should also be documented in the appropriate README(s) similar to other plugins that use secret store variables